app/assets/json/translations/react/en.json

```diff
@@ -829,6 +829,12 @@
   "listingDetails.applicationsDeadline": "Applications Deadline",
   "listingDetails.applicationsDeadline.timezone": "Pacific Time",
   "listingDetails.applicationsOpen": "Applications Open",
+  "listingDirectory.listingStatusContent.applicationDeadline": "Application deadline",
+  "listingDirectory.listingStatusContent.applicationsClosed": "Applications closed",
+  "listingDirectory.listingStatusContent.applicationsOpen": "Applications open",
+  "listingDirectory.listingStatusContent.lotteryResultsPosted": "Lottery results posted",
+  "listingDirectory.listingStatusContent.subContent.firstComeFirstServed": "First come, first served",
+  "listingDirectory.lotteryResultsPosted": "",
```

Comment:
issue: With these changes the image card status is not translated even for non FCFS listings. Should we continue to use the translated content until we get translations back?

Reply:
Good call!! Maybe I should just put this change behind a feature flag?

Reply:
Yea makes sense to me. I saw in [DAH-2588](https://github.com/SFDigitalServices/sf-dahlia-web/pull/2240/files) that we created a `see_the_unit` flag. Im ok with having a separate FCFS flag that way the see the unit feature can be released independently, once we get back translations for the content, without having to wait for the entire FCFS feature.